docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: "3"
+
+services:
+  git-badges-db:
```

Comment:
why not call it just `db`, there is only one after all

Reply:
It's just an example and it makes the sense of the service clearer.